frontend/packages/erd-core/src/features/erd/components/ERDContent/components/TableNode/TableDetail/Constraints/Constraints.tsx

```diff
@@ -0,0 +1,112 @@
+import type { Constraints as ConstraintsType } from '@liam-hq/db-structure'
+import { Check, Fingerprint, KeyRound, Link, Lock } from '@liam-hq/ui'
+import { clsx } from 'clsx'
+import type React from 'react'
+import { CollapsibleHeader } from '../CollapsibleHeader'
+import { CheckConstraintsItem } from './CheckConstraintsItem'
+import styles from './Constraints.module.css'
+import { ForeignKeyConstraintsItem } from './ForeignKeyConstraintsItem'
+import { PrimaryKeyConstraintsItem } from './PrimaryKeyConstraintsItem'
+import { UniqueConstraintsItem } from './Unique'
+
+type Props = {
+  constraints: ConstraintsType
+}
+
+export const Constraints: React.FC<Props> = ({ constraints: _constraints }) => {
+  const constraints = Object.values(_constraints)
+
+  const primaryKeyConstraints = constraints.filter(
+    (constraint) => constraint.type === 'PRIMARY KEY',
+  )
+  const foreignKeyConstraints = constraints.filter(
+    (constraint) => constraint.type === 'FOREIGN KEY',
+  )
+  const uniqueConstraints = constraints.filter(
+    (constraint) => constraint.type === 'UNIQUE',
+  )
+  const checkConstraints = constraints.filter(
+    (constraint) => constraint.type === 'CHECK',
+  )
+
+  const contentMaxHeight = constraints.length * 400
```

Comment:
I found [this comment](https://github.com/liam-hq/liam/pull/964/commits/aaa9f0d5e326cc899302a6580e9ca75fdaa7b666#r2011937342). I understand it doesn't have to be an accurate value but great enough to include everything.

Height of each table is smaller than 200px (1226.5px for ForeignKeyConstraintsItem table), so that 400px for each table is big enough to render everything, even considering titles and padding.

Reply:
@FunamaYukina Please review this line!

Reply:
Sorry for the lack of explanation. The `contentMaxHeight` is correct, as it is high enough to display the element.
If you would like to leave a comment, please do so below.🙏

https://github.com/liam-hq/liam/blob/7365e14af0bf7972a6f6992867d5cbca4d879505/frontend/packages/erd-core/src/features/erd/components/ERDContent/components/TableNode/TableDetail/Columns/Columns.tsx#L13-L14

Reply:
If you leave a comment, please merge this PR.👍

Reply:
Thanks for pointing it out. I left a comment here https://github.com/liam-hq/liam/pull/1329/commits/903b89843bd3cf6bc45d87767facf67c8ed301e2.